guides/06_other-tutorials/creating-a-chatbot.md

```diff
@@ -21,9 +21,13 @@ Make sure you have the `gradio` Python package already [installed](/quickstart).
 
 Let's get started! Here's how to build your own chatbot: 
 
-1. [Set up the Chatbot Model](#1-set-up-the-chatbot-model)
-2. [Define a `predict` function](#2-define-a-predict-function)
-3. [Create a Gradio Interface](#3-create-a-gradio-interface)
+- [How to Create a Chatbot](#how-to-create-a-chatbot)
```

Comment:
@dawoodkhan82 can you keep just the numbered bullets in here? Had it this way for SEO, to try and rank higher on the "How to build a chatbot" Query. Besides, doesn't really make sense to include the title, intro and prerequisites under "how to build your own chatbot" especially since this list is after the user has read them.